docs/Changelog.md

```diff
@@ -1,11 +1,14 @@
 # Changelog
 
-## 2.1.0 (in progress)
+The changelog list most feature changes between each release. The list is automatically created 
+based on merged pull requests. Search GitHub issues and pull requests for smaller issues. 
```

Comment:
I read through the changelog and deleted all items I think not belong here, but I guess it is subjective. Another read through is probably a good thing to do, please suggest more deletions/reversions.